### PR TITLE
Blog post about sensible defaults.

### PR DESCRIPTION
**Pull request by amjith**
@tsroten Whaddya think?

### CODE REVIEW

**Comment by j-bennet** [comment]
Good stuff @amjith. You might want to mention your recent `casing=auto` also. :)

I think there's one thing we didn't quite get with our sensible defaults (not mentioned in the article): decimal and float format specifiers. We left them blank by default, but it might have been a better solution to extract that information from user's locale settings.

File: content/sensible-defaults.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+default in pgcli. Every query is timed and the results are displayed at the
+bottom of the output.
+
+## History
+
+History in pgcli is unlimited. But pgcli has no option to separate the history
+for different databases. 
+
+The feature described in the blogpost talks about a cool feature to store the
+history of a sessio scoped to a database. This is an awesome feature that we
```
Comment (tsroten) [inline content/sensible-defaults.md]: Needs an `n` on `sessio`

:smile:

**Comment by tsroten** [comment]
@amjith This looks great! 👍 I'm happy with it as is, but I also think @j-bennet's suggestion is a good one. Maybe an "Additional Features" section at the bottom to talk about a couple other configs that Craig didn't mention?

Another possibility would be the sensible defaults for syntax colors, which is also overridable.

**Comment by amjith** [comment]
Fixed the typo. Thank you!


Regarding the expansion; I think it is best to keep the post short. I've changed the conclusion to say that we have a lot more config options and provided a link to the config file.

My hope is that the blogpost should make people intrigued and the motivated ones will checkout the rest of the config file which will further motivate them to try pgcli.

If we see good reception for this post I'll be happy to write a follow-up post that goes into a little bit more depth.

**Comment by amjith** [comment]
BTW, I don't know if you guys already saw this, but since pgcli.com uses netlify we can generate previews and the link to the preview is posted here: 

blog_post_about_sensible_defaults__by_amjith_ _pull_request__16_ _dbcli_pgcli_com

We should do the same for mycli.net soon.

**Review by tsroten — APPROVED** [review]
The preview is great :) I didn't notice that.

I'm fine with keeping it short.

**Comment by j-bennet** [comment]
What the what! I also didn't notice the preview! This is amazing!

@amjith I'd only add the `casing=auto` to your post then. Casing seems to be important to people. :)

**Review by j-bennet — APPROVED** [review]
Looks great!

**Comment by amjith** [comment]
I'll add the casing option and push a new commit.

**Pull request merged**